### DISCUSSION TITLE
Is it possible use this  sintaxis h-[300px] to set  dimention ?

### DISCUSSION THREAD

**Discussion by renaldodev** [tailwindlabs/tailwindcss#5225] [Help] [answered]
I saw on a Youtube channel where the guy defined the dimensions like this below and when I do just does not work.
## Example:
```Typescript
 <div className="relative  h-[300px] md:h-[500px]">
    ...
 </div>
```

**Comment by dance2die** [comment, marked as answer]
You need to enable a JIT (Just-in-Time) mode for a dynamic class declaration. 
https://tailwindcss.com/docs/just-in-time-mode

In your `tailwind.config.js`, you can enable it with `mode: 'jit'` option.

```js
  // tailwind.config.js
  module.exports = {
   mode: 'jit',
    purge: [
      // ...
    ],
    theme: {
      // ...
    }
    // ...
  }
```

**:warning: IMPORTANT**: You  need to configure the `purge` option (refer to the linked document).

**Reply by renaldodev** [reply]
thank you dude is very helpful
